controller/api/destination/server.go

```diff
@@ -353,6 +336,65 @@ func (s *server) GetProfile(dest *pb.GetDestination, stream pb.Destination_GetPr
 	return nil
 }
 
+// sendPodProfile accepts the GetProfile response stream, a pod and its port and
+// returns an error if it cannot stream a profile response back to the client. The
+// profile response sent here is different to a service's since it also includes
+// an endpoint.
```

Comment:
It would be nice to explicitly call out in the docstring that the pod parameter is optional.

Reply:
Great idea! I changed the text a bit to also make it more intuitive to figure out what the method is supposed to do.